### ISSUE TITLE
Go to definition not working

### ISSUE THREAD

**Issue by andrepaes** [elixir-lsp/coc-elixir#41]
It seems only go to definition not work here.

This is my configs:
<img width="573" alt="Screen Shot 2020-11-15 at 20 36 54" src="https://user-images.githubusercontent.com/12038420/99200177-8d9c6200-2782-11eb-9289-aa44d3a7a657.png">
<img width="558" alt="Screen Shot 2020-11-15 at 20 37 28" src="https://user-images.githubusercontent.com/12038420/99200181-912fe900-2782-11eb-9fe9-8705bb9c1f68.png">

It's very strange as the other features work properly but when i try to <C-]> it shows me a messages:

<img width="311" alt="Screen Shot 2020-11-15 at 20 40 07" src="https://user-images.githubusercontent.com/12038420/99200216-c4727800-2782-11eb-9a58-2781b4b8a402.png">

Can someone help me? Thanks in advance

**Comment by soundmonster** [comment]
Go to definition stopped working for me too, after I updated `elixir-ls` and `cos-elixir` to the latest versions on Nov 16.

**Comment by camilacremoneze** [comment]
I am not sure if this happens to u as well. I have different Elixir projects/versions and for some of them `go to definition` works, but for others does not. I have the same Andrepaes config and I have already rebuilt `.elixir_ls` following this thread https://github.com/elixir-lsp/coc-elixir/issues/1. Funny thing is that the projects in which `go to definition` works is not the same always, it seems it depends of the elixir version in which you compile `.elixir_ls`. Anyone can help me here? I have tried also to build an script to recompile `.elixir_ls` every time I access a different project, but it doesnt seem this is the way to go. Thanks.
